packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -623,8 +623,8 @@ class _DraggableSheetExtent {
     required bool snap,
     required List<double> snapSizes,
     required double initialSize,
-    Duration? snapAnimationDuration,
-    bool shouldCloseOnMinExtent = true,
+    required Duration? snapAnimationDuration,
+    required bool shouldCloseOnMinExtent,
```

Comment:
I guess whether these are `required` doesn't matter too much since it's a private class… I like how it looks though 👍

Reply:
private class should have required since we own all the call site